Packs/Armis/README.md

```diff
@@ -1,3 +1,5 @@
+Note: Support for this Pack will be moved to Partner starting October 26, 2022.
```

Comment:
I approve moving this to only 30 days into the future if Armis is as well. @marcellarichmond can you confirm w Armis plz?

Reply:
Updated

Reply:
Hi @mchasepan Can you please approve it?